app/assets/stylesheets/components/buttons.scss

```diff
@@ -1,109 +1,241 @@
 @import '../config/import';
 
-$outlined-border-width: 2px;
-
 // Basic styling
 .crayons-btn {
+  // Setup
+  --border-width: 0px;
+  --shadow-color: rgba(0, 0, 0, 0.05);
+  --vertical-padding: #{$su-2};
+  --horizontal-padding: #{$su-4};
```

Comment:
ooooo can you explain to me what is the difference between writing `#{$su-2}` versus just `$su-2`?

Reply:
`$...` is a SASS variable. When SASS variable is being used in CSS function that has brackets (for example `calc()`) you need this additional syntax. Another example that uses brackets is calling CSS variables: `property: var(--vertical-padding)` and that's why you need this extra syntax here.